gnirehtet.cmd

```diff
@@ -0,0 +1,149 @@
+@ECHO OFF
+CALL :locate_file ADB "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
+CALL :locate_file JAVA "%JAVA_HOME%\bin\java.exe"
+CALL :locate_file APK %~dp0gnirehtet.apk %~dp0app\build\outputs\apk\gnirehtet-release.apk %~dp0app\build\outputs\apk\gnirehtet-debug.apk
+CALL :locate_file RELAY %~dp0relay.jar %~dp0relay\build\libs\relay.jar
+
+::main
+IF NOT "%2"=="" SET serial="-s %2"
+CALL :do_%1 %serial% 2>NUL || CALL :do_help 
+GOTO :eof
+
+:do_help
+    ECHO Syntax: %~nx0 (install^|uninstall^|rt^|start^|stop^|relay^|killserver^|kill) ...
```

Comment:
Out of curiosity, why `%~nx0` here?

I found this [stackoverflow answer](http://stackoverflow.com/questions/5034076/what-does-dp0-mean-and-how-does-it-work/5034119#5034119) that says:

    %~nxI       - expands %I to a file name and extension only

But here there are no file involved.